org.KangLin.Rabbit.RemoteControl.json

```diff
@@ -0,0 +1,262 @@
+                    "type": "archive",
+                    "url": "https://www.libssh.org/files/0.11/libssh-0.11.1.tar.xz",
+                    "sha256": "14b7dcc72e91e08151c58b981a7b570ab2663f630e7d2837645d5a9c612c1b79"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DWITH_EXAMPLES=OFF"
+            ],
+            "cleanup": [
+                "/bin"
+            ]
+        },
+        {
+            "name": "libfuse",
+            "disabled": false,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libfuse/libfuse/archive/refs/tags/fuse-3.16.2.tar.gz",
+                    "sha256": "1bc306be1a1f4f6c8965fbdd79c9ccca021fdc4b277d501483a711cbd7dbcd6c"
+                }
+            ],
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": [
+                "-Dexamples=false",
+                "-Dutils=false"
+            ],
+            "cleanup": [
+            ]
+        },
+        {
+            "name": "FreeRDP",
+            "disabled": false,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FreeRDP/FreeRDP/archive/refs/tags/3.14.0.tar.gz",
+                    "sha256": "a7a979382703c2bc18aa9b22cfa12d7a441b6927ab45ac62ef8353786bc92241"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_VERBOSE_MAKEFILE=OFF",
+                "-DCMAKE_BUILD_TYPE:STRING=Release",
+                "-DCHANNEL_TSMF:BOOL=OFF",
+                "-DBUILD_TESTING:BOOL=OFF",
+                "-DWITH_MANPAGES:BOOL=OFF",
+                "-DWITH_WAYLAND:BOOL=OFF",
+                "-DWITH_GSSAPI:BOOL=OFF",
+                "-DWITH_SAMPLE:BOOL=OFF",
+                "-DWITH_OSS:BOOL=OFF",
+                "-DCHANNEL_URBDRC:BOOL=OFF",
+                "-DWITH_KRB5=ON"
+            ],
+            "cleanup": [
+                "/bin",
+                "/share"
+            ]
+        },
+        {
+            "name": "libvncserver",
+            "disabled": false,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/LibVNC/libvncserver/archive/refs/tags/LibVNCServer-0.9.15.tar.gz",
+                    "sha256": "62352c7795e231dfce044beb96156065a05a05c974e5de9e023d688d8ff675d7"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DWITH_OPENSSL=ON",
+                "-DWITH_GCRYPT=OFF",
+                "-DWITH_EXAMPLES=OFF",
+                "-DWITH_TESTS=OFF"
+            ],
+            "cleanup": [
+                "/bin/*",
+                "/lib/libvncserver.so*"
```

Comment:
why does it want to remove the DSO? What's the piint of building it?

Reply:
Only use static library.

Reply:
you should not use static libraries